test/CMakeLists.txt

```diff
@@ -2,290 +2,74 @@
 # Tests configuration
 #
 
-set(UVW_SRC_DIR ${CMAKE_SOURCE_DIR}/${PROJECT_SRC_DIR})
+set(UVW_SRC_DIR ${PROJECT_SRC_DIR})
+
+if(WIN32)
+    set(WINSOCK2 ws2_32)
+elseif(NOT APPLE)
+    find_library(LIBRT rt)
+endif()
 
 set(
     COMMON_LINK_LIBS
-    gtest
```

Comment:
[It's linked in `gtest_main`](https://github.com/google/googletest/blob/release-1.8.0/googletest/CMakeLists.txt#L92). The [example in googletest's README.md](https://github.com/google/googletest/blame/master/googletest/README.md#L175) uses it this way.